shared/docs/android/running.md

```diff
@@ -69,6 +69,18 @@ Same as below.
 `yarn rn-build-clean-android`
 
 
+## `$HOME/.../Android/sdk/ndk-bundle` Does not point to an Android NDK
+
+### macOS
+
+If you're hitting this issue, it is because you either don not have an NDK installed or installed an NDK with an older version of Android Studio that created an old directory path.
```

Comment:
```suggestion
If you're hitting this issue, it is because you either do not have an NDK installed or installed an NDK with an older version of Android Studio that created an old directory path.
```